app/controllers/tests_controller.rb

```diff
@@ -172,10 +172,10 @@ def has_authorization_with_test?
   end
 
   def test_available?
-     subject = Subject.fnd_by id: params[:subject_id]
-     return if subject.questions.count >= subject.question_amount
+    subject = Subject.find_by id: params[:subject_id]
+    return if subject.questions.count >= subject.question_amount
```

Comment:
hình như chị có cooment bên pull của hiệp nên pull này ko comment nữa
nhưng nhắc nhẹ, khi dùng những method có thể trả về nil thì em cần check xem kết quả  khác nil ko rồi đi  làm tiếp nhé hoặc dùng toán tử an toàn như try &. nhé